File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.py
# Last updated: 8/1/2025, 6:25:58 PM
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        even_digit_count = 0

        for num in nums:
            # Convert num to string and find its length
            length = len(str(num))
            if length % 2 == 0:
                even_digit_count += 1

        return even_digit_count

File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.java
// Last updated: 8/1/2025, 6:26:00 PM
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        if(nums.length!=0){
            String strNum;
            for (int i=0; i < nums.length;i++){
                strNum = Integer.toString(nums[i]);
                if (strNum.length() % 2 ==0)
                    count++;
            }
        }
        return count;
    }
}